### PR TITLE
rh-che-212: store logs of che-server in a persistent volume

### PR DESCRIPTION
**Pull request by benoitf**
It will require https://github.com/redhat-developer/rh-che/pull/229 before merge

Change-Id: Ifad2b7e9e30642c9f3e472568063f06093abc716
Signed-off-by: Florent BENOIT <fbenoit@redhat.com>

### CODE REVIEW

**Comment by benoitf** [comment]
@l0rd

File: apps/che/src/main/fabric8/deployment.yml [hunk 1]
```diff
@@ -16,6 +16,11 @@ spec:
             configMapKeyRef:
               key: "workspace-storage"
               name: "che"
+        - name: "CHE_LOGS_DIR"
+          valueFrom:
+            configMapKeyRef:
+              key: "che.logs.dir"
```
Comment (l0rd) [inline apps/che/src/main/fabric8/deployment.yml:22]: @benoitf I think that you need to add `che.logs.dir` to cm.yml too

**Review by l0rd — CHANGES_REQUESTED** [review]

File: apps/che/src/main/fabric8/deployment.yml (hunk 1 above)
Comment (benoitf) [reply apps/che/src/main/fabric8/deployment.yml:22]: Added

**Review by l0rd — APPROVED** [review]
Thank you @benoitf LGTM

**Comment by l0rd** [comment]
[merge]

**Pull request merged**